engines/python/setup/djl_python/properties_manager/trt_properties.py

```diff
@@ -24,11 +25,6 @@ class TensorRtLlmProperties(Properties):
     def validate_rolling_batch(cls, rolling_batch: str) -> str:
         rolling_batch = rolling_batch.lower()
 
-        if rolling_batch == RollingBatchEnum.disable.value:
```

Comment:
should we keep this in, but allow for t5 model to bypass? for every other model we require rolling batch, right?

Reply:
Yeah, but we need to look for a config file, and it can be in huggingface format or engine format, those logic exists in the tensorrt_llm_toolkit. We probably can't do it at the properties pydantic classes, maybe in handler. One thing we can do is to look for trtllm_python_backend property we populate in LmiConfigRecommender maybe.

Reply:
Now checking this and throwing error in handler. (Not in properties)